build.gradle

```diff
@@ -657,8 +658,8 @@ project('spring-integration-ws') {
 	description = 'Spring Integration Web Services Support'
 	dependencies {
 		compile project(":spring-integration-core")
-		compile "org.springframework:spring-oxm:$springVersion"
-		compile "org.springframework:spring-webmvc:$springVersion"
+		compile ("org.springframework:spring-oxm:$springVersion", optional)
+		compile ("org.springframework:spring-webmvc:$springVersion", optional)
 		compile ("org.springframework.ws:spring-ws-core:$springWsVersion") {
```

Comment:
I don't see reason to pollute `What's New`. 
Will add Migration Guide note though.

Although, we can restore them. Just because with the regular `spring-ws` they aren't `optional`.
However I really see cases with JMS, email, XMPP transports when we really don't need Web stuff, plus we may don't need marshalling/unmarhsalling for our SOAP requests.

But that is my opinion, so restoring...
(Or you can do that on merge :smile: )